content/blog/2024-12-03-tmil-11/index.md

```diff
@@ -0,0 +1,228 @@
+We'd love to [talk][] to people who are adopting Slang within the Rust ecosystem or interested in publishing Slang code for wider use.
+
+## Get Involved
+
+We welcome collaboration on any of our crates.
+This can include improving the documentation, implementing new features, improving our test coverage, or using them within your own code.
+
+We host an hour long office hours meeting each week where we discuss what's going on in our projects.
+See [#office hours in Zulip](https://xi.zulipchat.com/#narrow/channel/359642-office-hours) for details.
+
+* Daniel and Olivier's "office hours" appointments can still be booked by anyone for open-ended discussion of the ecosystem.
+  * [See Daniel's schedule here](https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ32eQYJ9DtZ_wJaYNtT36YioETiloZDIdImFpBFRo5-XsqGzpikgkg47LPsiHhpiwiQ1orOwwW2).
+  * [See Olivier's schedule here](https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ2t767ZRETD_TkRI_VxK2ZTG0VrO9OZ4l7HvTxefhtJcg85iK0ZN7zWNnAEZtH0Dn7C1GKxrmYM).
+
+[CSS Color Module Level 4]: https://www.w3.org/TR/css-color-4/
+[Low-level Piet]: https://xi.zulipchat.com/#narrow/channel/197075-gpu/topic/Low-level.20Piet
+[talk]: https://xi.zulipchat.com/#narrow/channel/197075-gpu
+[brendanzab]: https://github.com/brendanzab
+
+[parley#152]: https://github.com/linebender/parley/pull/743
+[parley#154]: https://github.com/linebender/parley/pull/743
+[parley#156]: https://github.com/linebender/parley/pull/743
+[parley#158]: https://github.com/linebender/parley/pull/743
+[parley#160]: https://github.com/linebender/parley/pull/743
+[parley#166]: https://github.com/linebender/parley/pull/743
+[parley#170]: https://github.com/linebender/parley/pull/743
+[parley#171]: https://github.com/linebender/parley/pull/743
+[parley#173]: https://github.com/linebender/parley/pull/743
+[parley#182]: https://github.com/linebender/parley/pull/743
+[parley#184]: https://github.com/linebender/parley/pull/743
+[resvg#843]: https://github.com/linebender/resvg/pull/843
+[resvg#850]: https://github.com/linebender/resvg/pull/850
+[vello#722]: https://github.com/linebender/vello/pull/722
+[vello#735]: https://github.com/linebender/vello/pull/735
+[vello#740]: https://github.com/linebender/vello/pull/740
+[vello#743]: https://github.com/linebender/vello/pull/743
+[xilem#670]: https://github.com/linebender/xilem/pull/743
+[xilem#731]: https://github.com/linebender/xilem/pull/743
+[xilem#732]: https://github.com/linebender/xilem/pull/743
+[xilem#734]: https://github.com/linebender/xilem/pull/743
+[xilem#735]: https://github.com/linebender/xilem/pull/743
+[xilem#736]: https://github.com/linebender/xilem/pull/743
+[xilem#742]: https://github.com/linebender/xilem/pull/743
+[xilem#749]: https://github.com/linebender/xilem/pull/743
+[xilem#754]: https://github.com/linebender/xilem/pull/743
+[xilem#755]: https://github.com/linebender/xilem/pull/743
+[xilem#762]: https://github.com/linebender/xilem/pull/743
```

Comment:
Copy-paste gone wrong here.
```suggestion
[parley#152]: https://github.com/linebender/parley/pull/152
[parley#154]: https://github.com/linebender/parley/pull/154
[parley#156]: https://github.com/linebender/parley/pull/156
[parley#158]: https://github.com/linebender/parley/pull/158
[parley#160]: https://github.com/linebender/parley/pull/160
[parley#166]: https://github.com/linebender/parley/pull/166
[parley#170]: https://github.com/linebender/parley/pull/170
[parley#171]: https://github.com/linebender/parley/pull/171
[parley#173]: https://github.com/linebender/parley/pull/173
[parley#182]: https://github.com/linebender/parley/pull/182
[parley#184]: https://github.com/linebender/parley/pull/184
[resvg#843]: https://github.com/linebender/resvg/pull/843
[resvg#850]: https://github.com/linebender/resvg/pull/850
[vello#722]: https://github.com/linebender/vello/pull/722
[vello#735]: https://github.com/linebender/vello/pull/735
[vello#740]: https://github.com/linebender/vello/pull/740
[vello#743]: https://github.com/linebender/vello/pull/743
[xilem#670]: https://github.com/linebender/xilem/pull/670
[xilem#731]: https://github.com/linebender/xilem/pull/731
[xilem#732]: https://github.com/linebender/xilem/pull/732
[xilem#734]: https://github.com/linebender/xilem/pull/734
[xilem#735]: https://github.com/linebender/xilem/pull/735
[xilem#736]: https://github.com/linebender/xilem/pull/736
[xilem#742]: https://github.com/linebender/xilem/pull/742
[xilem#749]: https://github.com/linebender/xilem/pull/749
[xilem#754]: https://github.com/linebender/xilem/pull/754
[xilem#755]: https://github.com/linebender/xilem/pull/755
[xilem#762]: https://github.com/linebender/xilem/pull/762
```

Reply:
Wow... That's an impressive error! Thanks